### PR TITLE
[13.0][MIG] base_multi_company

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
**base_multi_company** migration to v13

model 'res.company.assigment' has been removed, migration scripts will be made in product_multi_company and partner_multi_company.

cc ~ @LoisRForgeFlow @JordiBForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM :+1:

File: base_multi_company/models/multi_company_abstract.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    company_id = fields.Many2one(
+        string="Company",
+        comodel_name="res.company",
+        compute="_compute_company_id",
+        search="_search_company_id",
+    )
+    company_ids = fields.Many2many(
+        string="Companies",
+        comodel_name="res.company",
```
Comment (rousseldenis) [inline base_multi_company/models/multi_company_abstract.py:20]: @JoanSForgeFlow @JordiBForgeFlow I'm just wondering why the comodel name has been changed during migration to v13 and why not having done migration scripts ? Is there a reason ?

Thanks

Comment (rousseldenis) [reply base_multi_company/models/multi_company_abstract.py:20]: Ok, didn't see beginning comments

**Pull request merged**